cypress/integration/console/shared/api-keys/edit.spec.js

```diff
@@ -232,4 +242,62 @@ describe('API keys', () => {
       cy.findByRole('cell', { name: apiKeyName }).should('not.exist')
     })
   })
+
+  describe('User', () => {
```

Comment:
Please consider adding a simple test that verifies that the feature flag works as well.

Reply:
Just double-checked this. There is actually no reason to check for restrictions on creating personal api keys since all authorized users in the console have the `RIGHT_USER_SETTINGS_API_KEYS` right. Removed feature check.